### ISSUE TITLE
Requirements missing

### ISSUE THREAD

**Issue by AlmogCohen** [alisaifee/flask-limiter#35]
I installed the package of version 0.8.1 using `pip install` and the package was unable to run since the `limits` package was missing. I just added manually the `limits` package to my requirements.

 Can you fix it please...

**Comment by alisaifee** [comment]
Hi @AlmogCohen 

I just tested this by doing `pip install flask-limiter==0.8.1` in a fresh virtual env and `limits==1.0.8` was pulled in and installed as a dependency.

**Comment by AlmogCohen** [comment]
Cool, I'll check it out soon again and notify you if there will be any
problem.

**Comment by alisaifee** [comment]
Hi.  We're you able to confirm that the installation goes through?

**Comment by gcbirzan** [comment]
The problem is that the egg on pypi is broken, it contains the limits package 'inline'.

**Comment by alisaifee** [comment]
Hi @gcbirzan, thanks.  I'm not sure what you mean by inline? `limits` is specified in `setup.py`.

**Comment by gcbirzan** [comment]
I mean it's in the egg file, just download it and see. There's a limits.py, and all the stuff from the limits package inside the flask-limiter directory.

**Comment by alisaifee** [comment]
@gcbirzan : Ah, I see what you mean, good catch - I had a dirty build directory (I'll remove them for the next release). Having said that, those extra files are not in the install manifest and are therefore not installed in `site-packages` when the package is installed.

```
ls -1 lib/python2.7/site-packages/flask_limiter/*.py
lib/python2.7/site-packages/flask_limiter/__init__.py
lib/python2.7/site-packages/flask_limiter/_version.py
lib/python2.7/site-packages/flask_limiter/errors.py
lib/python2.7/site-packages/flask_limiter/extension.py
lib/python2.7/site-packages/flask_limiter/util.py
lib/python2.7/site-packages/flask_limiter/version.py
```

**Comment by gcbirzan** [comment]
We're using buildout and it seems to install it. Also, any ETA on the next release?

**Comment by alisaifee** [comment]
I've made a `0.8.2` release. No functional changes - just a cleaned up egg/tarball.

**Comment by gcbirzan** [comment]
Much appreciated, I'll test it tomorrow morning (EU) and let you know!

**Comment by alisaifee** [comment]
you're welcome. I'm going to close this specific issue for now as I don't see any problem with the dependency on `limits`

**Comment by alisaifee** [comment]
Seems like there is indeed a problem when installing via tarball

**Comment by alisaifee** [comment]
Fixed as per #38
